tests/conftest.py

```diff
@@ -52,6 +52,10 @@ def polars_constructor(obj: Any) -> IntoDataFrame:
     return pl.DataFrame(obj)
 
 
+def polars_lazy_constructor(obj: Any) -> pl.LazyFrame:
```

Comment:
I think I couldn't use `IntoDataFrame` here. What would be the correct way to type this?

Should we add a `IntoLazyFrame` type too?

Reply:
yup, probably! and maybe we will also need `IntoFrame` for when you don't know?

on the typing front, I'd like to see if we can make `DataFrame` generic